### DISCUSSION TITLE
Welcome to `SatGPT` Discussions! Intros go here 👇

### DISCUSSION THREAD

**Discussion by lalligagger** [lalligagger/satgpt-app#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ivandorte** [comment]
Hi 🤘I'm Ivan, from Italy, actually living not so far from the eternal city of Rome. I have a MSc degree in Geology and GIS. For several years I worked as a GIS tech (GIS/Python/PostgreSQL) in the utilities industry (public sector) in the beautiful city of Siena (Tuscany).  I also had a brief, unexciting experience at ESRI Italia and the University of Rome.
I spend much of my time learning new geo-python related stuff, trekking, picking mushrooms/asparagus  and preparing pizza 🍕, which I love so much! Recently, I discovered Panel and other interesting frameworks like Solara, which allows you to build powerful dashboards and visualization. I generally tend to avoid lego-like tools like ArcGIS dashboards and MS Power BI, because I don't have full control on what I'm doing and are not accessible to everyone.
Actually, I'm looking for  an interesting opportunity in the GIS/EO industry. Planning to move in North Italy/EU27 by the end of the year. What I will do in this community? Sharing my ideas and talk about python tools and stuff related to GIS/EO industry. Also get in touch with people interested in these topics.

**Comment by lalligagger** [comment]
Hi Ivan, really appreciate your work so far and thanks for kicking this off! It's too bad we didn't meet virtually a bit sooner - I just had my first visit to Rome to present at an ESA/ ESRIN conference on thermal EO. :) Hopefully I'll be back.

My background is primarily in satellite imaging systems engineering. I started with a physics degree at Montana State University in Bozeman and followed up with a Masters in optics/ engineering management. Since then I have worked mostly on new space/ smallsat projects but also spent some time at Project Kuiper working on laser comm terminals.

I am now at Hydrosat working on a constellation for daily land surface temperature mapping at field scale (10s of meter resolution). It's been a great chance to absorb the latest and greatest of the geo python & STAC ecosystems, which my colleagues are the real experts in. SatGPT has been a fun project to spin together this recent experience with data access/ analysis and all that's happening in the LLM world. Looking forward to sharing with a wider audience, getting some feedback, and seeing where folks are interested in taking it!
